.git_hooks/pre-commit

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env sh
```

Comment:
This seems weird, is there a reason you didn't use `/bin/sh`?
```suggestion
#!/bin/sh
```

Reply:
Not a major difference for most cases, but using `/usr/bin/env` will take `sh` from the environment used while `/bin/sh` assumes this is the path to the shell installation which is common but not mandatory.

Especially if the environment has multiple shell versions, for example if `PATH` contains "/path_to_sh/:/bin".

`/user/bin/env sh` will take `sh` from `path_to_sh` which is first in `PATH` while `/bin/sh` will use `/bin/sh`.

I fine either way, let me know.

Reply:
> common but not mandatory

I think posix requires `/bin/sh`. But I don't feel strongly about it, it's fine to keep it as-is.

Reply:
Mentioning POSIX is a good way to get me to double check myself 😊 .

https://pubs.opengroup.org/onlinepubs/009695399/utilities/sh.html

> Applications should note that the standard PATH to the shell cannot be assumed to be either /bin/sh or /usr/bin/sh, 
> and should be determined by interrogation of the PATH returned by getconf PATH , ensuring that the returned 
> pathname is an absolute pathname and not a shell built-in.